cloudinit/distros/networking.py

```diff
@@ -114,6 +177,9 @@ class BSDNetworking(Networking):
     def is_physical(self, devname: DeviceName) -> bool:
         raise NotImplementedError()
 
+    def settle(self, *, exists=None) -> None:
+        raise NotImplementedError()
```

Comment:
@goneri @igalic I have a specific and a general question around this. Specifically: is there an equivalent to [`udevadm settle`](https://man7.org/linux/man-pages/man8/udevadm.8.html) on *BSD? If there isn't then I'll convert this to a `pass`, if there is and it's trivial, I can include it here; otherwise, you can propose it as a follow-up. :-)

Generally: I realised while writing this that raising `NotImplementedError` in these implementations may convert some "nothing happens on *BSD" error conditions into "we definitely raise `NotImplementedError` on BSD" error conditions, which could stop cloud-init's execution (whereas previously we would have failed to run $cmd on BSD but continued on regardless, for example). Do you think that we should actually be `pass`ing by default in these implementations (with some sort of comment that indicates that this is a "we haven't implemented this yet" `pass`, not a "there's nothing to do for this on BSD" `pass`)?

Reply:
i don't see anything in [`devd(8)`](https://www.freebsd.org/cgi/man.cgi?devd) or [`devctl(8)`](https://www.freebsd.org/cgi/man.cgi?devctl) that would would wait for events to settle…

…so let's simply assume that `devd(8)` is extremely instant, and so we can either use pass or `sleep(1)`

Reply:
I've made it `pass`, thanks for the input!